cli/tests/e2e/targets/dependency_aware/poetry_quoted_key/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
```

Comment:
do you want to test the arbitrary comment removal part of the manifest file too?